app/forms/EducationQuestionnaireForm.scala

```diff
@@ -93,20 +93,19 @@ object EducationQuestionnaireForm {
 
       def getOptionalUniversityList(isCandidateCivilServant: String): List[Question] = {
```

Comment:
i think there is not explicit checks on isCandidateCivilServant hence can be removed